marlowe-playground-client/src/Wallet.purs

```diff
@@ -413,7 +413,7 @@ handleAction StartContract = do
 handleAction ResetContract = do
   assign _slot zero
   assign (_contracts <<< traversed <<< _started) false
-  assign (_contracts <<< traversed <<< _marloweState) (NEL.singleton (emptyMarloweState zero))
+  assign (_contracts <<< traversed <<< _marloweState) (NEL.singleton (emptyMarloweStateWithSlot zero))
```

Comment:
do we ever use `emptyMarloweStateWithSlot` with a slot other than `zero` these days? Perhaps we don't need it any more?